sdk/cosmos/.azure-pipelines/cosmos.test.yml

```diff
@@ -26,27 +26,67 @@ jobs:
       vmImage: $(OSVmImage)
 
     steps:
+      - task: PowerShell@1
+        displayName: 'Download Public Cosmos Emulator'
+        inputs:
+          scriptType: inlineScript
+          inlineScript: |
+            Write-Host "Downloading Cosmos Emulator - $(EmulatorMsiUrl)"
+            wget "$(EmulatorMsiUrl)" -outfile "$env:temp\azure-cosmosdb-emulator.msi"
```

Comment:
May I ask why we are now consuming this via an msi?

Reply:
Evidently there are problems using the emulator with v3 currently, and it cannot be used. This process is copied from https://cosmos-db-sdk-public.visualstudio.com/cosmos-db-sdk-public/_build?definitionId=32